### PR TITLE
chore(release): prepare 8.8.0

### PR DESCRIPTION
**Pull request by robertohuertasm**
@KingDarBoja, you should add your email or/and your website to the package.json! 

Welcome to the team! ❤️

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+- Enhancement: Updated `vscode file icons` with `stable` and `insider` versions (1.35). ([@KingDarBoja](https://github.com/KingDarBoja) in [#2092](https://github.com/vscode-icons/vscode-icons/pull/2092))
+- Fix: `Haxelib` icon should only be used for `.haxelib`. ([@Gama11](https://github.com/Gama11) in [#2089](https://github.com/vscode-icons/vscode-icons/pull/2089))
+- Feature: Support for `ClojureScript`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2086](https://github.com/vscode-icons/vscode-icons/pull/2086))
+- Feature: Support for `Dojo`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2085](https://github.com/vscode-icons/vscode-icons/pull/2085))
+- Feature: Support for `Azure` folder. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2081](https://github.com/vscode-icons/vscode-icons/pull/2081))
+- Feature: Support for `SSO` folder. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2080](https://github.com/vscode-icons/vscode-icons/pull/2080))
+- Feature: Support for `Nearly`. ([@Floffah](https://github.com/Floffah) in [#2077](https://github.com/vscode-icons/vscode-icons/pull/2077))
```
Comment (JimiC) [inline CHANGELOG.md]: It's `Nearley` not `Nearly`.

Comment (robertohuertasm) [reply CHANGELOG.md]: fixed

**Comment by JimiC** [comment]
LGTM. Only waiting for @KingDarBoja to provide the info for `package.json`.

**Comment by KingDarBoja** [comment]
> @KingDarBoja, you should add your email or/and your website to the package.json!
> 
> Welcome to the team! ❤️

Thank you, I am so happy of being part of `vscode-icons` team 💯 

Also updated the `package.json` file as requested. My website is not complete atm (been lazy due to job) but it's ok.

**Pull request merged**